workspaces/report-portal/package.json

```diff
@@ -37,9 +37,9 @@
     "directory": "workspaces/report-portal"
   },
   "devDependencies": {
-    "@backstage/cli": "^0.27.0",
+    "@backstage/cli": "^0.27.1",
     "@backstage/e2e-test-utils": "^0.1.1",
-    "@backstage/repo-tools": "^0.9.5",
+    "@backstage/repo-tools": "^0.9.7",
     "@changesets/cli": "^2.27.1",
     "@spotify/prettier-config": "^15.0.0",
     "concurrently": "^8.0.0",
```

Comment:
```suggestion
```

and the `dev` script has also changed: https://backstage.github.io/upgrade-helper/?from=1.30.2&to=1.31.3